concent-builder/jenkins-configuration-files/config.xml

```diff
@@ -0,0 +1,84 @@
+  </actions>
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+    <com.coravy.hudson.plugins.github.GithubProjectProperty plugin="github@1.29.3">
+      <projectUrl>https://github.com/bartoszbetka/concent-deployment.git/</projectUrl>
+      <displayName></displayName>
+    </com.coravy.hudson.plugins.github.GithubProjectProperty>
+    <org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
+      <triggers>
+        <com.cloudbees.jenkins.GitHubPushTrigger plugin="github@1.29.3">
+          <spec></spec>
+        </com.cloudbees.jenkins.GitHubPushTrigger>
+      </triggers>
+    </org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
+  </properties>
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition" plugin="workflow-cps@2.61">
+    <script>properties([[$class:
+&apos;GithubProjectProperty&apos;,
+    displayName: &apos;&apos;,
+    projectUrlStr: &apos;https://github.com/bartoszbetka/concent-deployment.git/&apos;],
+pipelineTriggers([githubPush()])])
+
+node {
+    stage (&apos;Checkout repository&apos;) {
+        dir(&apos;/home/jenkins/concent-deployment/&apos;) {
+            git url:
+            &apos;https://github.com/bartoszbetka/concent-deployment.git&apos;,
+            branch: &apos;dev&apos;
+        }
+        ansiblePlaybook( 
+            inventory: &apos;/home/jenkins/inventory&apos;, 
+            playbook: &apos;/home/jenkins/concent-deployment/concent-builder/set-concent-version.yml&apos;
+       )
+    }
+    stage (&apos;Build&apos;) {
+        ansiblePlaybook( 
+            extraVars: [
+                cluster: &apos;concent-dev&apos;,
+            ],
+            inventory: &apos;/home/jenkins/inventory&apos;, 
+            playbook: &apos;/home/jenkins/concent-deployment/concent-builder/install-repositories.yml&apos;
+        )
+        ansiblePlaybook( 
+            extraVars: [
+                cluster: &apos;concent-dev&apos;,
+            ],
+            inventory: &apos;/home/jenkins/inventory&apos;, 
+            playbook: &apos;/home/jenkins/concent-deployment/concent-builder/build-test-and-push.yml&apos;
+        )
+    }
+    stage (&apos;Deploy&apos;) {
+        ansiblePlaybook( 
+            extraVars: [
+                cluster: &apos;concent-dev&apos;,
+            ],
+            inventory: &apos;/home/jenkins/inventory&apos;, 
+            playbook: &apos;/home/jenkins/concent-deployment/concent-builder/deploy.yml&apos;
+        )
+    }
+    stage (&apos;Test&apos;) {
+        ansiblePlaybook( 
+            extraVars: [
+                cluster: &apos;concent-dev&apos;,
+            ],
+            inventory: &apos;/home/jenkins/inventory&apos;, 
+            playbook: &apos;/home/jenkins/concent-deployment/concent-builder/configure-and-run-tests.yml&apos;
+        )
+    }
+}</script>
+    <sandbox>true</sandbox>
+  </definition>
+  <triggers/>
+  <disabled>false</disabled>
+</flow-definition>
```

Comment:
Why you want 4-character here ? This is default configuration file from jenkins.

Reply:
Because we want this file to be edited by us with a text editor (not automatically regenerated using Jenkins GUI) and we use 4-space indents everywhere (except formats like YAML where syntax rules do not let us).

It's fine to keep the default config in a separate commit but please then refactor it in another commit.